hw/ip/otbn/data/otbn.hjson

```diff
@@ -119,13 +119,8 @@
             The error cause if an error occurred.
 
             Software should read this register before clearing the err
-            interrupt to avoid race conditions.
-
-            Possible values:
-
-            - 0x0 (ErrCodeNoError):     No error occurred.
-            - 0x1 (ErrCodeBadDataAddr): Load or store to invalid address
-            - 0x2 (ErrCodeCallStack):   Call stack underflow/overflow
+            interrupt to avoid race conditions. See 'Error conditions' in the
+            main OTBN documentation for the possible values.
```

Comment:
Maybe add a hyperlink here? And this text appears in the main OTBN documentation, so I don't think you need to say that.

Reply:
Given this is used by reggen I didn't know what else might consume the doc string. If it's purely our hugo doc generator then the hyperlink should be workable, if something else wants to consume it it could look odd. Also because it's reggen it'll have to be a brittle relative path that would break if we alter something about the doc paths without remembering to update this.

Reply:
We've already got markdown magic in the file. I don't think there's a problem adding more.